app/assets/stylesheets/visualizations.css.scss

```diff
@@ -213,8 +103,25 @@ body[data-page-name='visualizations/show'] .visualizations-controller,
     color: $hint;
   }
 
+  .i {
+    font-style: italic;
```

Comment:
`.i`? This is bad.  Can we call this something that doesn't sound like an index? Also, why do we need this as opposed to just using `<i>` tag?